src/css/styles/_helpers.scss

```diff
@@ -35,33 +35,15 @@
   padding-left: 0.3em;
 }
 
-.h-current-lang {
-  color: $col-txt-gray-light !important;
-}
-
-.l-container-padding {
-  padding-right: 4rem;
-  padding-left: 4rem;
-}
-
-.c-section {
-  padding-bottom: 4rem;
-}
-
-.l-last-element {
-  margin-bottom: 3rem;
-}
-
-.l-center-y {
-  position: absolute;
-  top: 50%;
-  transform: translateY(-50%);
+.h-btn-divider {
+  padding: 0.6rem;
+  margin: 0.6rem 1.2rem 0.6rem 0;
+  font-family: $font-praktika-bold;
+  font-size: 0.6rem;
+  text-transform: uppercase;
+  letter-spacing: 0.1em;
 }
 
-.l-col-clean {
-  padding: 0;
-}
-
-.l-col-padding {
-  padding: 0.5rem;
+.h-current-lang {
+  color: $col-txt-gray-light !important;
 }
```

Comment:
IMHO `.h-light`, `.h-black` is not used

- [ ] remove `.h-light`
- [ ] remove `.h-black`